spring-web-modules/spring-web-url/src/test/java/com/baeldung/spring/slash/SlashParsingControllerIntTest.java

```diff
@@ -0,0 +1,87 @@
+package com.baeldung.spring.slash;
+
+import static org.junit.Assert.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.net.URI;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+@AutoConfigureMockMvc
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+public class SlashParsingControllerIntTest {
```

Comment:
This test fails because you didn't copy corresponding com.baeldung.spring.slash package.

Reply:
Fixed the test cases